app/common/locales/en.json

```diff
@@ -760,6 +760,13 @@
         }
     },
     "data_import" :{
+        "see_imported_posts" : "See imported posts",
+        "import_another_csv" : "Import another CSV file",
+        "import_complete" : "Your import is complete",
+        "complete_info" : "The data from your CSV spreadsheet, <em>{{filename}}</em>, was successfully imported into your <a href=\"/settings/survey/{{form_id}}\">{{form_name}}</a> survey.",
+        "import_another" : "Import another CSV file",
+        "nearly_complete" : "Your import is nearly complete",
+        "finish_info" : "The data from your CSV spreadsheet is still being processed. You're welcome to leave this page or wait until the process is complete. Either way, we'll notify you when your data is ready.",
```

Comment:
So, angularjs $resource does not automatically abort requests, it can be forced to on either a timeout or a promise resolution. In this case, the API won't know the client hungup until it sends the response.

This seems to be a somewhat contentious issue as some people argue that promises should never be cancellable. Looking at the angularjs code it does appear that they only send an abort if a timeout is defined. At least, this is what I've found in the code and they have no documentation that suggests otherwise.

I would argue that as much as an async call is still a connection between two endpoints it is conceptually a separation where the response is not dependent on the requester not hanging up.

So not to prolong this - I'll commit and merge and this can be revisisted - but the "we will notify you bit" is I think important though perhaps it should be "If you remain in the app we will send you a notification to say the import has completed"